cmake/CMakeLists.txt

```diff
@@ -414,18 +468,21 @@ if("${libTKernelExt}" STREQUAL ".a")
     if (NOT APPLE)
         set(LIB_RT "rt")
     endif()
-    set(OPENCASCADE_LIBRARIES ${OPENCASCADE_LIBRARIES} ${OPENCASCADE_LIBRARIES} ${OPENCASCADE_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT} ${LIB_RT} dl)
```

Comment:
Hi Johannes, many thanks for your pr. Can you elaborate on this particular change? IIRC duplicating the libraries was necessary (at least on some versions of gcc) to make sure static libraries where properly searched for all (possibly cyclic) dependencies.